compile.sh

```diff
@@ -76,12 +76,46 @@ OPTS+=" -Dhadoopversion=$HADOOP_VERSION"
 OPTS+=" -Dsparkversion=$SPARK_VERSION"
 OPTS+=" $PLAY_OPTS"
 
-set -x
-trap "exit" SIGINT SIGTERM
 
 project_root=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 cd ${project_root}
 
+cd ${project_root}
+
+
+#if npm is installed, install bower,ember-cli and other components for new UI
```

Comment:
Can you make a note of this in the documentation?

Also, at least until we deprecate the older UI, can we make the compilation of the new UI optional?